src/persist.js

```diff
@@ -31,6 +40,12 @@ exports.save = async function (processed) {
         { transaction }
       )
       await Promise.all(rows.map(async (row) => {
+        const hash = crypto.createHash('sha256')
+        hash.update(`${row.first_name} ${row.last_name} ${row.dob}`)
```

Comment:
```hash.update(JSON.stringify({first_name: row.first_name, last_name: row.last_name, dob: row.dob}))```

Reply:
Ignore the above. I forgot to sort the object keys (to guarantee a stable order), but instead you should do: `hash.update(JSON.stringify([row.first_name, row.last_name, row.dob]))`

oh and you also need to move up the code that checks for a missing `dob` field (and sets it to null), otherwise your hashes won't always match

Reply:
And lastly, did you update the code that queries the DB for FN+LN+DOB to use the hash instead?